crates/core/src/writer/stats.rs

```diff
@@ -1,7 +1,3 @@
-use std::sync::Arc;
```

Comment:
while still on nightly i think, `rustfmt` has an [opinion](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#group_imports) - i.e. the prior ordering would be the idiomatic one.